src/new-storage/DeleteExpiredCmd.js

```diff
@@ -0,0 +1,139 @@
+const cassandra = require('cassandra-driver')
+const fetch = require('node-fetch')
+const pLimit = require('p-limit')
+
+const validateConfig = require('../helpers/validateConfig')
+
+class DeleteExpiredCmd {
+    constructor(config) {
+        validateConfig(config)
+
+        this.baseUrl = config.streamrUrl
+
+        const authProvider = new cassandra.auth.PlainTextAuthProvider(config.cassandraNew.username, config.cassandraNew.password)
+        this.cassandraClient = new cassandra.Client({
+            contactPoints: [...config.cassandraNew.hosts],
+            localDataCenter: config.cassandraNew.datacenter,
+            keyspace: config.cassandraNew.keyspace,
+            authProvider,
+            pooling: {
+                maxRequestsPerConnection: 32768
+            }
+        })
+
+        // used for limited concurrency
+        this.limit = pLimit(5)
```

Comment:
just curious why 5?

Are any of the cassandra calls significantly faster/slower than the rest? Wondering if you should have different queues for the different request types so slow operations don't block fast ones.

Reply:
it's super easy to kill `EE` with a lot of request, that's why I put 5 to reduce the load